### ISSUE TITLE
Absolute package path in import stmt prevents vendoring

### ISSUE THREAD

**Issue by andy-maier** [Phynix/yamlloader#20]
In the pywbem project, we are trying to vendor the yamlloader package.

This failed due to the absolute import of yamlloader in yamlloader/ordereddict/loaders.py and yamlloader/ordereddict/dumpers.py:
```
 import yamlloader.settings
```

In other places, submodules of yamlloader are imported using a relative package path, which works with vendoring, for example in yamlloader/ordereddict/__init__.py:
```
from .loaders import ...
```

This can be fixed by changing the absolute package path to a relative package path in yamlloader/ordereddict/loaders.py and yamlloader/ordereddict/dumpers.py:
```
 from .. import settings
```
and adjusting the use of this submodule accordingly.

For completeness, the error we get is:
```
python -c "import pywbem"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/runner/work/pywbem/pywbem/pywbem/__init__.py", line 49, in <module>
    from ._cim_operations import *  # noqa: F403,F401
  File "/home/runner/work/pywbem/pywbem/pywbem/_cim_operations.py", line 166, in <module>
    from ._recorder import LogOperationRecorder
  File "/home/runner/work/pywbem/pywbem/pywbem/_recorder.py", line 36, in <module>
    from ._vendor import yamlloader
  File "/home/runner/work/pywbem/pywbem/pywbem/_vendor/yamlloader/__init__.py", line 12, in <module>
    from . import ordereddict
  File "/home/runner/work/pywbem/pywbem/pywbem/_vendor/yamlloader/ordereddict/__init__.py", line 5, in <module>
    from .loaders import Loader, SafeLoader, CLoader, CSafeLoader
  File "/home/runner/work/pywbem/pywbem/pywbem/_vendor/yamlloader/ordereddict/loaders.py", line 9, in <module>
    import yamlloader.settings
ModuleNotFoundError: No module named 'yamlloader'
```

**Comment by andy-maier** [comment]
Addressed in PR #21

**Comment by mayou36** [comment]
Thanks for the fix, a new version is released (1.1.0)
feel free to close

**Comment by andy-maier** [comment]
Thanks for the very quick release!
